app/views/obligation/AppealByLetterKickOutPage.scala.html

```diff
@@ -0,0 +1,63 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+
+@this(layout: Layout,
+        bullets: components.bullets,
+        link: components.link,
+        pageHeader: components.pageHeader,
+        govukInsetText : GovukInsetText)
+
+@(pageMode: PageMode)(implicit userRequest: UserRequest[_], messages: Messages, appConfig: AppConfig)
+
+@layout(pageTitle = messages("appealByLetter.headingAndTitle"),
+        isPageFullWidth = false,
+        optPageMode = Some(pageMode)) {
+    @pageHeader(messages("appealByLetter.headingAndTitle"))
+    <p class="govuk-body">@messages("appealByLetter.p1")</p>
+    @bullets(
+        Seq(
+            Html(messages("appealByLetter.li.1.1")),
+            Html(messages("appealByLetter.li.1.2")),
+            Html(messages("appealByLetter.li.1.3"))
+        )
+    )
+    <p class="govuk-body">@messages("appealByLetter.p2")</p>
+    @bullets(
+        Seq(
+            Html(messages("appealByLetter.li.2.1")),
+            Html(messages("appealByLetter.li.2.2"))
+        )
+    )
+
+    <div class="govuk-inset-text">
+        <p class="govuk-body govuk-!-margin-bottom-0">@messages("appealByLetter.inset.1")</p>
+        <p class="govuk-body govuk-!-margin-bottom-0">@messages("common.address.1")</p>
+        <p class="govuk-body govuk-!-margin-bottom-0">@messages("common.address.2")</p>
+        <p class="govuk-body govuk-!-margin-bottom-0">@messages("common.address.3")</p>
```

Comment:
I like this way of doing it, in other pages we use `<br>` which doesn't seem right... no changes needed here :)